### ISSUE TITLE
Proguard mapping file not included inside debug_meta on flutter application

### ISSUE THREAD

**Issue by lberaldodev** [getsentry/sentry#77949]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

1. Flutter application (3.19.6)

2. Sentry 4.2.0

3. Build.gradle settings 
`includeProguardMapping = true
 autoUploadProguardMapping = true
 dexguardEnabled = true
 uploadNativeSymbols = true
 autoUploadNativeSymbols = true
 includeNativeSources = true
 includeSourceContext = true`

4. Check if proguard file was uploaded
![Image](https://github.com/user-attachments/assets/f65025bd-bfa3-430a-a144-2c97dfc4b4de)

5. Check if debug files includes proguard file
![Image](https://github.com/user-attachments/assets/ec356ba0-a92a-4fde-9133-d719a0840b0a)

6. Check the json issue 
![Image](https://github.com/user-attachments/assets/9923ba79-91cf-447e-a100-a0fc1453b978)

### Expected Result

When we upload the proguard mapping file, we expect to see the full stack trace 
![Image](https://github.com/user-attachments/assets/7966eab5-ac23-4f32-9af0-524af5185348)

### Actual Result

No stacktrace included and no `proguard` type included on `debug_meta`
![Image](https://github.com/user-attachments/assets/8e2e0e21-3290-4b0d-9d56-ea503354319f)

### Product Area

Issues - Source Maps

**Comment by scttcper** [comment]
Does the issue have the release set?

**Comment by lberaldodev** [comment]
> Does the issue have the release set?

Yes, the release is set to each proguard correctly. We can inspect the release oppening  the link from the image below
![Image](https://github.com/user-attachments/assets/bd07f52f-fe4e-49d4-af5f-f59d8067cdb7)

**Comment by scttcper** [comment]
I mean does the issue have a release tag

**Comment by lberaldodev** [comment]
> I mean does the issue have a release tag

My bad :)  I don't think so...

It's a very old problem here at my project, he already used fastlane , sentry cli, dart flutter cli, and now we are using gradle.  I think that is not a version problem... but i'm focused to solve it.

**Comment by scttcper** [comment]
I think this is usually why sourcemaps wouldn't work correctly. If that doesn't fix your issue, feel free to reopen this issue.

**Comment by lberaldodev** [comment]
Hey @scttcper my bad. Is this tag that you mentioned? :"D 
![Image](https://github.com/user-attachments/assets/36705b7d-e74b-41f8-b0a6-fd17548c765c)

**Comment by scttcper** [comment]
@lberaldodev yes does that release have the proguard mappings associated with it?

**Comment by scttcper** [comment]
@lberaldodev can you share a link to the issue

**Comment by lberaldodev** [comment]
I can share the json issue, can be it?

**Comment by scttcper** [comment]
@lberaldodev opening a ticket with support would also allow us to debug further. Link to this issue as well if you do that.

**Comment by mrduncan** [comment]
I'm going to close this issue to tidy up for now - happy to reopen if someone can share a repro case or a support ticket ID with more detailed information
